concepts/05 Widgets/Form/01 Getting Started with Form/40 Organize the Form Items/20 In Groups.md

```diff
@@ -0,0 +1,206 @@
+You can divide form items in groups. To do this, declare items with the 'group" [itemType](/Documentation/ApiReference/UI_Widgets/dxForm/Item_Types/GroupItem/#itemType) in the [items[]](/Documentation/ApiReference/UI_Widgets/dxForm/Configuration/#items) array. A group can contain [simple items](/Documentation/ApiReference/UI_Widgets/dxForm/Item_Types/SimpleItem/), other groups, [tabs](/Documentation/ApiReference/UI_Widgets/dxForm/Item_Types/TabbedItem/tabs/) or [empty items](/Documentation/ApiReference/UI_Widgets/dxForm/Item_Types/EmptyItem/). You can configure each group's layout separately. 
+
+If you use modular approaches and have more than one item type in a Form, declare the simple items as in the example below.
+
+The folowing code shows two groups in a Form with two columns. Each group occupies a separate column, and the `Personal Information` group has two columns inside it:
+
+---
+##### jQuery
+
+    <!-- tab: index.js -->
+    $(function() {
+        $("#formContainer").dxForm({
+            formData: {
+                name: "John Heart",
+                position: "CEO",
+                hireDate: new Date(2012, 4, 13),
+                officeNumber: 901,
+                phone: "+1(213) 555-9392",
+                skype: "jheart_DX_skype",
+                email: "jheart@dx-email.com",
+                notes: "John has been in the Audio/Video industry since 1990."
+            },
+            colCount: 2,
+            items: [{
+                itemType: "group",
+                caption: "Personal Information",
+                colCount: 2,
+                items: ["name", "position", "hireDate", "officeNumber", "notes"]
```

Comment:
Can we show one and the same set of fields in each topic and add new to show new feature?

["name", "position", "hireDate", "officeNumber"] ?

Reply:
Good idea, thanks.